regression.py

```diff
@@ -89,6 +89,45 @@ def testADF_ADF2013_01_dvb_gopt_b_unconverged_adfout(logfile):
     """An unconverged geometry optimization to test for empty optdone (see #103 for details)."""
     assert hasattr(logfile.data, 'optdone') and not logfile.data.optdone
 
+# def testADF_ADF2013_01_stopiter_dvb_sp_adfout(logfile):

```

Comment:
Yeah, have them run and skip the problematic things. Just add comments to explain what's going on.